### PR TITLE
feat(collector-list-item): apply collector item design

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- apply job status icons & change the label text
image

- apply truncate at plugin name (name: 75% / version: 25%)
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 1]
```diff
@@ -59,52 +59,47 @@
...
-                    />
-                </span>
-                <span class="icon-fill-wrapper">
-                    <p-i
-                        name="ic_check"
-                        class="check-icon"
-                        height="1rem"
-                        width="1rem"
-                        color="inherit"
+                <div v-for="(jobItems, index) in TEMP_JOB_STATUS"
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:62]: praise: 👍

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 2]
```diff
@@ -11,8 +11,10 @@
                             width="1.25rem"
                             height="1.25rem"
                 />
-                <span class="plugin-name">{{ props.item.plugin.name }}</span>
-                <span class="plugin-version">v{{ props.item.plugin.info.version }}</span>
+                <div class="plugin-info">
+                    <span class="plugin-name">{{ props.item.plugin.name }}</span>
+                    <span class="plugin-version">v{{ props.item.plugin.info.version }}</span>
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:16]: issue: It seems that `props.item` is sometimes undefined. Would it be better to use a computed property instead?

**Review by sulmoJ — APPROVED** [review]
I left a comment!
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue [hunk 3]
```diff
@@ -97,7 +97,7 @@ const state = reactive({
     infoItems: [
         { key: COLLECTOR_ITEM_INFO_TYPE.PLUGIN, label: 'Plugin' },
         { key: COLLECTOR_ITEM_INFO_TYPE.STATUS, label: 'Current Status' },
-        { key: COLLECTOR_ITEM_INFO_TYPE.JOBS, label: 'Recent Collector Jobs' },
+        { key: COLLECTOR_ITEM_INFO_TYPE.JOBS, label: 'Recent Collector Jobs (Last 5 days)' },
         { key: COLLECTOR_ITEM_INFO_TYPE.SCHEDULE, label: 'Schedule' },
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue:101]: question: They are not translated. is it okay?

**Review by WANZARGEN — APPROVED** [review]
LGTM~ I left a small question!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue (hunk 3 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue:101]: The columns on the existing collector page have not been translated. Let me confirm!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue (hunk 2 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue]: check it again plz!!!

**Pull request merged**